kumascript/macros/ListGroups.ejs

```diff
@@ -54,26 +54,24 @@ for(let name of groupNames) {
   // Go through the tags and build badges if there are any to add
 
   if (page && page != undefined) {
-    let tags = page.tags;
-
-    if (tags) {
```

Comment:
FYI check on existence not needed as discussed in https://github.com/mdn/yari/pull/7828/files#r1052753810 - so removed it.